roles/core/nfs_server/molecule/default/verify.yml

```diff
@@ -0,0 +1,69 @@
+---
+- name: Verify
+  hosts: all
+  tasks:
+
+    - name: Collect package facts
+      package_facts:
+        manager: auto
+
+    - name: Collect services facts
+      service_facts:
+
+    - name: Collect services of zone public
+      command: firewall-cmd --zone=public --list-all
+      register: firewall_cmd_result
+      changed_when: False
+
+    - name: Check rpc-bind and nfs are in zone public
+      assert:
+        that: firewall_cmd_result.stdout |
+                    regex_findall(("rpc-bind|nfs"), multiline=False) | length == 2
+
+    - name: Assert nfs-utils and rpcbind are installed
+      assert:
+        that: "'{{ item }}' in ansible_facts.packages"
+      loop:
+        - nfs-utils
+        - rpcbind
+
+    - name: Retrieve file /etc/exports system status
+      stat:
+        path: /etc/exports
+      register: register_stat
+
+    - name: Assert file /etc/exports exist
+      assert:
+        that: "register_stat.stat.exists"
+
+    - name: Retrieve /opt/software in /etc/exports file
+      lineinfile:
+        path: /etc/exports
+        regexp: "^/opt/software"
+        state: absent
+      check_mode: yes
+      register: export_sw_result
```

Comment:
Can you add `changed_when: false` so the verify won't report changes in the output?

Reply:
Done in commit: 5cb76a02a40198f54bb9aab67f55e46f1f43850b